# Introduction to Manual Curation - HiC and JBrowse
#BGA24/sessions #GitPod #Generating_a_reference/3

This session is part of [**Biodiversity Genomics Academy**](https://thebgacademyBGA23.org/)

[![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#https://github.com/thebgacademy/manual-curation) [![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/thebgacademy/manual-curation) 
> ![note] For BGA, please use the GitPod Environment!
> If using the GitHub Codespace, fork to your own account first
## Session Leader(s)

Genome Reference Informatics Team, Wellcome Sanger Institute

- Jo Wood
- Joanna Collins
- Dominic Absolon
- Tom Mathers
- Michael Paulini
- Camilla Santos

## Description

By the end of this session you will be able to:

1. Understand the Manual curation pipeline
2. Understand how Jbrowse can be utilised as a tool both to  aid curation and also for post curation analysis
3. Interpret a HiC Map
4. Understand how to curate an assembly using PretextView and associated curation scripts
5. Create an updated/curated assembly fasta

## Prerequisites

1. Understanding the terms genome assembly, reads, contigs
2. Understanding of linux command line basics
3. To have read/watched the rapid curation documentation available at <https://gitlab.com/wtsi-grit/rapid-curation/-/tree/main>
4. To have read through the Interpreting_HiC_maps guide
5. Have access to a 3 button mouse

>[!warning] "Please make sure you MEET THE PREREQUISITES and READ THE DESCRIPTION above"
>
>    You will get the most out of this session if you meet the prerequisites above.
>
>    Please also read the description carefully to see if this session is relevant to you.
>    
>    If you don't meet the prerequisites or change your mind based on the description or are no longer available at the session time, please email tol-training at sanger.ac.uk to cancel your slot so that someone else on the waitlist might attend.
