---
title: Aligning whole genomes using Cactus
aliases:
  - cactus
date_created: 18_30-04-2024
data_modified: 18_30-04-2024
search_name: Alignments with Cactus
---
#BGA23/sessions #Workshop #Cactus #Alignments #Tools 

> [!caution] This session may now be out of date so beware!!!
> This session is part of  [[tags/BGA23]]

## Session Leader(s)

Chiara Bortoluzzi, ETH ZÃ¼rich, Switzerland  
Thiago A.L. Genez, European Bioinformatics Institute, United Kingdom

## Description
[![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#https://github.com/BGAcademy23/cactus)
GitHub:: https://github.com/BGAcademy23/cactus

By the end of this session you will be able to:

1. Run cactus yourself
2. Understand how the cactus algorithm works 
3. Understand what input data are required in cactus
4. Get familiar with some downstream analyses that require a multiple sequence alignment

## Prerequisites

- Expertise with linux command line basics
- Understanding of basic concepts of multiple sequence alignments

> [!warning] !!! warning "Please make sure you MEET THE PREREQUISITES and READ THE DESCRIPTION above"
> 
> You will get the most out of this session if you meet the prerequisites above.
> 
> Please also read the description carefully to see if this session is relevant to you.
> 
> If you don't meet the prerequisites or change your mind based on the description or are no longer available at the session time, please email tol-training at sanger.ac.uk to cancel your slot so that someone else on the waitlist might attend.
