---
title: gfastar - a tool suite to assist genome assembly
aliases:
  - gfastar
date_created: 18_30-04-2024
data_modified: 18_30-04-2024
search_name: GFastar
---
#BGA23/sessions #Tools #Gfastar #QC #Workshop

> [!caution] This session may now be out of date so beware!!!
> This session is part of  [[tags/BGA23]]

## Session Leader(s)

Giulio Formenti  
Research Assistant Professor, The Rockefeller University

Cassidy Johnson  
Graduate Fellow, The Rockefeller University

Jack Medico
Graduate Fellow, The Rockefeller University

## Description
[![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#https://github.com/BGAcademy23/gfastar)
GitHub:: https://github.com/BGAcademy23/gfastar

By the end of this session you will be able to:

1. Compute genome assembly summary statistics using gfastats
2. Manipulate genome assembly graphs using gfastats
3. Compute QV estimates using kreeq
4. Evaluate genome assemblies at scale using Buttercup

## Prerequisites

1. Understanding basic terms in genome assembly, such as reads, contigs, scaffolds, ...
2. Basic command line experience.

> [!warning] !!! warning "Please make sure you MEET THE PREREQUISITES and READ THE DESCRIPTION above"
> 
> You will get the most out of this session if you meet the prerequisites above.
> 
> Please also read the description carefully to see if this session is relevant to you.
> 
> If you don't meet the prerequisites or change your mind based on the description or are no longer available at the session time, please email tol-training at sanger.ac.uk to cancel your slot so that someone else on the waitlist might attend.
> 
