---
title: Introduction to NextFlow for genomics
aliases:
  - nextflow-genomics
date_created: 18-30_04_2024
data_modified: 18_30-04-2024
search_name: Nextflow in Genomics
---

#BGA23/sessions #Pipeline #Nextflow #NF-Core #Workshop


> [!caution] This session may now be out of date so beware!!!
> This session is part of  [[tags/BGA23]]

## Session Leader(s)

Solenne Correard, Resarch associate for the Canada Biogenome Project

I would like to acknowledge that I am living and working on the traditional, ancestral, and unceded territory of the Musqueam, Squamish and Tsleil-Waututh people.

Affiliation : Canada's Michael Smith Genome Sciences Centre, BC Cancer Research Centre



I did my PhD in France studying the dog genome (mostly the non-coding part), followed by a post-doc on the Silent Genomes Project, a project aiming to reduce health care disparities and improve diagnostic success for children with rare genetic diseases from Indigenous populations in Canada. I am now leading the bioinformatic analysis for the Canadian BioGenome Project.

## Description
[![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#https://github.com/BGAcademy23/nextflow-genomics)
GitHub:: https://github.com/BGAcademy23/nextflow-genomics

The session will last for about 1h30, and will be split in 2 parts of about 40 min each (we'll take a 10 min break in the middle).

Each part will include an introduction, some coding and hopefully many questions and discussion!

Outline of the session :

- Nextflow (Part 1) : What is it? When should I use it? How do I use it? 

Practice : Writing your first Nextflow pipeline to generate a contig assembly. 

- NF-core (Part 2): What is it? When should I use it? How do I use it? 

Practice : Use NF-core tools to assess the quality of the generated assembly. 

- Conclusion: Resources to learn more about Nextflow and NF-core and how to take part in the community.

By the end of this session you will be able to:

1. Read a nextflow pipeline and modify it
2. Write a basic nextflow pipeline
3. Use a nf-core pipeline
4. How to join the nextflow / nf-core community

The outline may change if I come up with a better idea while generating the material!

## Prerequisites

1. Expertise with linux command line basics (`cd`, `mv`, `rm`, `wget`, `curl` etc)
2. Understanding of bash scripts, and the need for bioinformatics workflows
3. Understand the general role and input/output files of the following tools : [hifiasm HiFi only](https://hifiasm.readthedocs.io/en/latest/pa-assembly.html), [Quast](https://github.com/ablab/quast), [multiQC](https://multiqc.info)

> [!warning] !!! warning "Please make sure you MEET THE PREREQUISITES and READ THE DESCRIPTION above"
> 
> You will get the most out of this session if you meet the prerequisites above.
> 
> Please also read the description carefully to see if this session is relevant to you.
> 
> If you don't meet the prerequisites or change your mind based on the description or are no longer available at the session time, please email tol-training at sanger.ac.uk to cancel your slot so that someone else on the waitlist might attend.
